docs/source/examples/Natural and artificial perturbations.mystnb

```diff
@@ -81,18 +88,15 @@ def f(t0, state, k):
     return du_kep + du_ad
 
 
-rr = propagate(
-    orbit,
-    tofs,
-    method=cowell,
-    f=f,
-)
+rr, _ = orbit.to_ephem(
+    EpochsArray(orbit.epoch + tofs, method=CowellPropagator(f=f)),
+).rv()
```

Comment:
This is how users are expected to transition from `propagate` to `Orbit.to_ephem`.